keycloak/themes/eddi/login/resources/css/eddi-login.css

```
@@ -0,0 +1,735 @@
+        --eddi-recess:        #f5f5f4;
+        --eddi-inset-shadow:  inset 0 1px 2px rgba(28, 25, 23, .06);
+        --eddi-card-sheen:    none;
+        --eddi-card-edge:     inset 0 1px 0 0 rgba(255, 255, 255, .9);
+        --eddi-card-shadow:   0 10px 30px -12px rgba(28, 25, 23, .18);
+
+        /* The wordmark ships as a white monochrome PNG. Inverting it produces
+           black on the same transparency — invert() leaves the alpha channel
+           alone — which is why no second asset is needed. */
+        --eddi-logo-filter:   invert(1);
+    --eddi-chevron:       url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 10 6'%3E%3Cpath fill='%2378716c' d='M0 0h10L5 6z'/%3E%3C/svg%3E");
+        --eddi-chevron:       url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 10 6'%3E%3Cpath fill='%2378716c' d='M0 0h10L5 6z'/%3E%3C/svg%3E");
```

Stylelint (17.14.1)

[error] Duplicate custom property "--eddi-chevron" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)